lib/lichess.py

```diff
@@ -355,10 +353,11 @@ def challenge(self, username: str, payload: REQUESTS_PAYLOAD_TYPE) -> JSON_REPLY
         """Create a challenge."""
         return self.api_post("challenge", username, payload=payload, raise_for_status=False)
 
-    def cancel(self, challenge_id: str) -> JSON_REPLY_TYPE:
+    def cancel(self, challenge_id: str) -> None:
         """Cancel a challenge."""
-        return self.api_post("cancel", challenge_id, raise_for_status=False)
+        self.api_post("cancel", challenge_id, raise_for_status=False)
 
+    # TOOD: Replace with a more specific type
```

Comment:
This comment hasn't been removed.

Reply:
Whoops, sorry, was just doing a search and missed this